README.md

```diff
@@ -13,6 +13,15 @@ While the Android client integrates with the system with regular update checks
 and notifications, this is a command line client that talks to connected
 devices via [ADB](https://developer.android.com/tools/help/adb.html).
 
+### Go dependency
+
+Example of go setup on debian based systems:
+
+    sudo apt-get install golang-go
+    # put these in a shell startup file to persist them:
+    export GOPATH=$HOME/go
+    PATH="$GOPATH/bin:$PATH"
+
```

Comment:
I agree. I had some logic for including it, but reviewing things again, I changed my mind.